### PR TITLE
adding mailroom part 1

### PR DESCRIPTION
**Pull request by Cemito**

### CODE REVIEW

File: students/cem/mailroom/mailroom.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+        response = input(prompt)  # continuously collect user selection
+        # now redirect to feature functions based on the user selection
+        if response == "list":
+            list_donors()
+        elif response == "1":
+            send_thank_you()
+        elif response == "2":
+            create_report()
+        elif response == "3":
+            exit_program()
```
Comment (majstorpoki) [inline students/cem/mailroom/mailroom.py:42]: break is much easier

File: students/cem/mailroom/mailroom.py [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+        
+        Regards,
+        Cem
+        """.format(response, donated, CHARITY_NAME))
+
+
+def send_thank_you():
+    """Send thank you e-mails to existing or new donors"""
+    while True:
+        response = input("Enter the name of who you would like to send a thank you e-mail to "
```
Comment (majstorpoki) [inline students/cem/mailroom/mailroom.py:74]: q should also be listed on options.

File: students/cem/mailroom/mailroom.py [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+    header_total_given = "Total Given"
+    header_num_gifts = "Num Gifts"
+    header_average_gift = "Average Gift"
+
+    header = f"{header_donor_name:<25} | {header_total_given:>10} | {header_num_gifts:>10} | {header_average_gift:>10}"
+
+    print(header)
+    print(len(header)*'-')
+
+    for key, value in donors.items():
```
Comment (majstorpoki) [inline students/cem/mailroom/mailroom.py:107]: Reports should be listed in the order of total donations.

**Review by majstorpoki — APPROVED** [review]
Very good 14/15

File: students/cem/mailroom/mailroom.py (hunk 3 above)
Comment (Cemito) [reply students/cem/mailroom/mailroom.py:107]: I've fixed this for Mailroom pt2.

File: students/cem/mailroom/mailroom.py (hunk 2 above)
Comment (Cemito) [reply students/cem/mailroom/mailroom.py:74]: I've also added this as an option in mailroom pt2.

**Pull request merged**